bank-accounts.rb

```diff
@@ -35,14 +39,42 @@ def self.find(id)
       end
     end
 
+    def self.master_list # Accounts and their respective owners in one big happy array
+      master_list = []
+      account_owners_csv = CSV.read("./support/account_owners.csv")
+
+      account_owners_csv.each do |row|
+        account = Bank::Account.find(row[0].to_i)
+        account_owner = self.find(row[1].to_i)
+
+        account_owner.accounts[:account] = account
```

Comment:
I'm not sure if this is an appropriate use of hash since it seems like this could be assigned directly to an account variable rather than setting it with a key/value pair